src/inference/tests/functional/ov_extension_test.cpp

```diff
@@ -291,13 +274,6 @@ class CustomReLU : public ov::op::Op {
 };
 
 #if defined(ENABLE_OV_IR_FRONTEND)
-TEST_F(OVExtensionTests, ReshapeIRWithOldExtension) {
```

Comment:
I am not a fan of removing tests on legacy API if we still have this API. Let's just separate these tests to the separate file or do nothing.